samples/graphene-redis/redis-manifest.json

```diff
@@ -1,10 +1,7 @@
 {
     "Packages": {
         "redis-package": {
-            "Debug": true,
-            "SignerID": "43361affedeb75affee9baec7e054a5e14883213e5a121b67d74a0e12e9d2b7a",
-            "ProductID": 13,
-            "SecurityVersion": 1
+            "Debug": true
```

Comment:
```suggestion
            "Debug": true,
            "SignerID": "43361affedeb75affee9baec7e054a5e14883213e5a121b67d74a0e12e9d2b7a",
            "ProductID": 13,
            "SecurityVersion": 1
 ```

Reply:
I changed this, because I had issues with the activation of the marble. From the documentation I thought that if `debug` is set, then the other values don't matter. This seems not to be the case. 
> set to true if the enclave is to be run in debug mode. This allows you to experiment with deploying your application with Marblerun without having to worry about setting correct values for the above properties, but note that enclaves in debug mode are not secure.
I had an issue there: https://github.com/edgelesssys/marblerun/blob/787f8458008197a41437bf38a3bb392ef95fa6e7/coordinator/quote/ert.go#L44
Maybe I just had issue because I removed the last lines in the graphene manifest because there was no MARBLERUN comment preceeding them. Will check that on tomorrow.

Reply:
If debug is true, the other values can be omitted. If it is false, the other values must be defined.
If they are defined, they are always checked, regardless of whether debug is true or false.
You had issues because you did not sign with the private key that has this signerid.

Reply:
Why do we hardcode some value for SignerID there if it's not embedded in the Dockerfile? I am confused.